back-end/hub-core/src/main/java/io/apicurio/hub/core/integrations/ApicurioEventType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.hub.core.integrations;
+
+/**
+ * @author Fabian Martinez
+ */
+public enum ApicurioEventType {
+
+    DESIGN_CREATED,
+    DESIGN_UPDATED,
+    DESIGN_DELETED;
+
+    private String cloudEventType;
+
+    private ApicurioEventType() {
+        this.cloudEventType = "io.apicurio.hub."+this.name().toLowerCase().replace("_", "-");
```

Comment:
Suggestion, maybe add a field to the enum and use that instead of replacing it this way?

Reply:
I agree - that would be a nice improvement.

Reply:
Although thinking about it more, the result is pretty constrained - it's not like we don't have control of the result and may get something unexpected.  I'm 50/50 on this.  Whatever Fabian decides is fine. :)